include/aspect/utilities.h

```diff
@@ -192,6 +192,21 @@ namespace aspect
                                                       double phi );   //longitude (radians)
 
     /**
+     * A struct to enable numerical output with a comma as thousands separator
+     */
+    struct ThousandSep : std::numpunct<char>
+    {
+      char do_thousands_sep() const
+      {
+        return ',';
+      }
+      std::string do_grouping() const
+      {
+        return "\3";  // groups of 3 digits
```

Comment:
I see now what this is supposed to mean. Maybe write it as `"\003"` to make it clearer that this is an octal number?